lib/jira/commands/assign.rb

```diff
@@ -2,8 +2,9 @@ module Jira
   class CLI < Thor
 
     desc "assign", "Assign a ticket to a user"
+    method_option :assignee, aliases: "-a", type: :string, default: nil, lazy_default: "auto", banner: "ASSIGNEE"
```

Comment:
Sorry about that, I misunderstood what the `aliases` flag meant. Was pretty tired when I review this!